Managers/TimerMan.cpp

```diff
@@ -50,19 +59,34 @@ namespace RTE {
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 	long long TimerMan::GetAbsoluteTime() const {
+#ifdef _WIN32
 		LARGE_INTEGER tickReading;
 		QueryPerformanceCounter(&tickReading);
 		long long ticks = tickReading.QuadPart;
-
-		return (ticks * 1000000) / m_TicksPerSecond;
+#elif __unix__
+    timespec my_TimeSpec;
```

Comment:
Whitespacing here is bad, please realign stuff and make sure to use tabs not spaces. Applies below too.